osidb/tasks.py

```diff
@@ -0,0 +1,61 @@
+from celery.utils.log import get_task_logger
+from django.conf import settings
+from django.contrib.contenttypes.models import ContentType
+from django.db.models import OuterRef, Q, Subquery
+from django.db.models.functions import Cast
+
+from config.celery import app
+from osidb.core import set_user_acls
+from osidb.mixins import Alert
+
+logger = get_task_logger(__name__)
+
+
+@app.task(autoredtry_for=(Exception,))
```

Comment:
There seems to be an extra `d` in the parameter name. Also I am thinking what is the reason and consequence of the retry. What errors do we expect? I would probably not expect any exception being thrown during a regular run of the cleanup. If there is some, that might rather be some really serious fatality or a bug. Wouldn't that retry then actually mask the error? Maybe even repeating the failure indefinitely? If so then the retry might actually be undesirable and we should just let it fail so we can see the failure in Flower (not sure about Splunk).

Reply:
Honestly, I added it because I used the previous tasks as an example and they had it, but yeah it does not make much sense as it should not throw an error on any case, and even then, since is just removing old data we can afford some occasional errors.

I just removed it